packages/tracer/tests/unit/Tracer.test.ts

```diff
@@ -133,19 +133,19 @@ describe('Class: Tracer', () => {
       expect(putAnnotation).toBeCalledWith('Service', 'foo');
 
     });
+    test('when called when a serviceName has not been set in the constructor or environment variables, it adds the default service name as an annotation', () => {
 
-    test('when called while a serviceName has not been set, it does nothing', () => {
-
+      process.env = { ...ENVIRONMENT_VARIABLES, POWERTOOLS_SERVICE_NAME : undefined };
```

Comment:
Curious as to why you changed this, the `POWERTOOLS_SERVICE_NAME` env variable is set elsewhere before the tests are run, so deleting it equals to setting it to `undefined`.

I'm not against the change per-se, however in other instances it's done with the `delete`.

Reply:
I'll use `delete`